cookbook/entities/acl.rst

```diff
@@ -0,0 +1,246 @@
+    user find your entity.
+
+.. _coobook-entities-acl-create:
+
+Creating Access Control Lists
+-----------------------------
+
+You have to options to define your custom access control lists:
+
+.. _cookbook-entity-acl-controller:
+
+#. In your controller class, you can use the ``@Acl`` annotation:
+
+   .. code-block:: php
+       :linenos:
+
+       // src/AppBunde/Controller/TaskController.php
+       namespace AppBundle\Controller;
+
+       use AppBunde\Entity\Task;
+       use Oro\Bundle\SecurityBundle\Annotation\Acl;
+       use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+       use Symfony\Component\HttpFoundation\Request;
+
+       class TaskController extends Controller
+       {
+           /**
+            * @Acl(
+            *   id="app_task_view",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="VIEW"
+            * )
+            */
+           public function indexAction()
+           {
+               // ...
+           }
+
+           /**
+            * @Acl(
+            *   id="app_task_create",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="CREATE"
+            * )
+            */
+           public function createAction(Request $request)
+           {
+               // ...
+           }
+
+           /**
+            * @Acl(
+            *   id="app_task_edit",
+            *   type="entity",
+            *   class="AppBundle:Task",
+            *   permission="EDIT"
+            * )
+            */
+           public function editAction(Task $task, Request $request)
+           {
+               // ...
+           }
+       }
+
+   Using the ``@Acl`` annotation does not only create new access control lists to which you can
+   refer in other parts of your code it will also trigger the access decision manager when your
+   actions are accessed by users and thus protect them from being accessed without the needed
+   permissions.
+
+#. If you do not want to protect any controller methods or if you prefer to keep the definition of
+   your ACLs separated from the application code, you can define them using some YAML config in a
+   file named ``acl.yml``:
+
+   .. code-block:: yaml
+       :linenos:
+
+       # src/AppBunde/Resources/config/acl.yml
+       app_task_create:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: CREATE
+
+       app_task_delete:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: DELETE
+
+       app_task_edit:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: EDIT
+
+       app_task_view:
+           type: entity
+           class: AppBunde\Entity\Task
+           permission: VIEW
+
+.. _coobook-entities-acl-check:
+
+Perform Access Checks
+---------------------
+
+Once you have configured the ACLs you can protect all parts of your application:
```

Comment:
Please add the description that developer can check access via securiy facade service in the code.